pkg/apis/kf/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -281,6 +281,46 @@ func (in ObjectReferences) DeepCopy() ObjectReferences {
 	return *out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
What caused this generated code to exist?

Reply:
It's been removed. I'm not actually sure why that was generated, however when I re-ran the generate scripts, it was deleted.